README.md

```diff
@@ -28,6 +28,7 @@ Because of possible binary incompatibilities here are the dependency versions us
 |         0.2        |     3.1.1     |     0.6.0    |
 |         0.3        |     3.1.1     |     0.7.0    |
 |         0.4        |     3.1.1     |     0.8.x    |
+|         0.5-M1     |     3.2.0-M2  |     0.8.x    |
```

Comment:
This should be made more clear in the README itself but this README is generated from `src/main/tut/README.md`. Changes should be made there, and then after `tut` is run, a new (and verified to compile) README is generated here.